### ISSUE TITLE
Peel off pre-release duct tape

### ISSUE THREAD

**Issue by lukevmorris** [influxdata/chronograf#1227]
https://github.com/influxdata/chronograf/blob/master/ui/src/dashboards/components/CellEditorOverlay.js#L120

`ResizeContainer` and `ResizeBottom` are calculating heights and wiring those values through to deeper components. During the DE flip refactor, the wire from ResizeContainer to Visualization was cut somewhere, and it was pretty hard to figure out what specifically was going wrong.

I wonder if we can move to a simpler system:
```jsx
<ResizeContainer>
  <QueryBuilder />
  <Visualization />
</ResizeContainer>
```
might spit out:

```html
<div class="resize-container">
  <div class="query-builder" style="height: 60%"></div>
  <div class="resize-handle"></div>
  <div class="visualization" style="height: 40%"></div>
</div>
```

And
```jsx
<ResizeContainer>
  <Visualization />
  <div>
    <OverlayControls />
    <QueryBuilder />
  </div>
</ResizeContainer>
```

might spit out
```html
<div class="resize-container">
  <div class="visualization" style="height: 60%"></div>
  <div class="resize-handle"></div>
  <div style="height: 40%">
    <div class="overlay-controls"></div>
    <div class="query-builder></div>
  </div>
</div>
```

And that's all it should do. Heights of the sub components (i.e. `Visualization`, `QueryBuilder`, `OverlayControls`) would be controlled by a stylesheet, since they're different depending on the context (i.e. `DataExplorer` and `CellEditorOverlay`). And `Visualization` (or one of the components nested within) would be responsible for determining its current height and feeding that value to `Dygraph`.

**Comment by alexpaxton** [comment]
I'm a big fan

**Comment by jaredscheib** [comment]
Looks like this may have been resolved by https://github.com/influxdata/chronograf/pull/1385 -- is that the case, @alexpaxton ?

**Comment by alexpaxton** [comment]
@jaredscheib ah yeah good catch, def fixed by #1385
